platform/security/common/src/test/java/ddf/security/assertion/impl/SecurityAssertionImplTest.java

```diff
@@ -102,6 +103,9 @@ public void testSampleAssertion() throws Exception {
     assertEquals(NUM_ATTRIBUTES, assertion.getAttributeStatements().size());
     assertEquals(NUM_NAUTH, assertion.getAuthnStatements().size());
     assertEquals(
```

Comment:
I chose not to assert on value because the value is stored in the file `saml.xml`. I didn't want to tie the test code too tightly to the file. However, I see that the other assertion do use values in the files. If you think it is important, I'll be happy to add an assertion on the value.